_posts/2016/2016-05-25-spring-io.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Spring I/O 2016"
+date: "2016-05-25 09:00:00"
+image: blog-header/springio.jpg
+categories: events
+authors: ["Xavi"]
+---
+Last 19th and 20th of May a conference called *Spring I/O* took place in Barcelona. It was the 5th edition of an event whose goal is to spread the word about the well-known framework [Spring](https://spring.io/). Luckily, four *epagees* including myself had the chance to attend the conference.
+
+The conference itself consisted of a number of talks and workshops, up to four at the same time. Each day began with a keynote and, as it was bound to be, there was plenty of coffee available, although considering there were lots of Java developers I think tea would have fit like a glove.
+
+As you might expect, I didn't find all activities equally interesting or fun, so I'll just talk about those I enjoyed the most in no particular order.
+
+# "Cloud Native Java", by Josh Long
+My colleague Mathias spoke about this same talk on his [post related to GeeCON Kraków](https://developer.epages.com/blog/2016/05/20/geecon.html), so I won't repeat what he said, but that was undoubtedly one of the highlights of the conference. In fact, a Josh Long ([@starbuxman](https://twitter.com/starbuxman)) quote was included in the official Spring I/O t-shirt: "Make Jar not War".
+
+# "Securing RESTFul services with Spring HATEOAS and Hdiv", by Roberto Velasco
+Roberto Velasco ([@hdivroberto](http://twitter.com/hdivroberto)) is the CEO of [Hdiv](https://www.hdivsecurity.com/), whose goal is to provide security solutions for applications. During his talk he pointed out the fact that it's very important to implement security measures in both client and server sides. For instance, given a certain form, preventing users from entering wrong or unexpected values into its fields is not good enough because those security measures could be bypassed. Therefore, such fields should be double-checked in the server side, and Mr. Velasco explained how do they do it in [Hdiv](https://www.hdivsecurity.com/).
```

Comment:
Therefore, such fields should be double-checked in the server side > Therefore, such fields should be double-checked on the server side